.buildkite/scripts/buildkite-integration-tests.sh

```diff
@@ -15,6 +15,11 @@ if [ -z "$TEST_SUDO" ]; then
   exit 1
 fi
 
+if [ "${FIPS:-false}" == "true" ]; then
```

Comment:
Why is this needed to just run go test ? Test code should also be FIPS-capable ?

Reply:
Using `GOEXPERIMENT=systemcrypto` forces a check for microsoft/go